hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/types/ClusterStateResponse.java

```diff
@@ -198,8 +198,8 @@ public Builder setBuckets(long buckets) {
       return this;
     }
 
-    public void setDeletedKeys(long deletedKeys) {
-      this.deletedKeys = deletedKeys;
+    public void setkeysAwaitingDeletion(long keysAwaitingDeletion) {
```

Comment:
```suggestion
    public void setKeysAwaitingDeletion(long keysAwaitingDeletion) {
```

Reply:
@smengcl Completed Changes suggested in Review